pkg/pod/container.go

```diff
@@ -113,6 +113,91 @@ func (builder *ContainerBuilder) WithSecurityCapabilities(sCapabilities []string
 	return builder
 }
 
+// WithDropSecurityCapabilities drops SecurityCapabilities from the container definition.
+//
+//nolint:gocognit
+func (builder *ContainerBuilder) WithDropSecurityCapabilities(sCapabilities []string, redefine bool) *ContainerBuilder {
+	glog.V(100).Infof("Dropping a list of SecurityCapabilities %v from container %s",
+		sCapabilities, builder.definition.Name)
+
+	if !areCapabilitiesValid(sCapabilities) {
+		glog.V(100).Infof("Given SecurityCapabilities %v are not valid. Valid list %s",
+			sCapabilities, AllowedSCList)
+
+		builder.errorMsg = "one of the give securityCapabilities is invalid. Please extend allowed list or fix parameter"
+	}
+
+	var sCapabilitiesList []v1.Capability
+	for _, capability := range sCapabilities {
+		sCapabilitiesList = append(sCapabilitiesList, v1.Capability(capability))
+	}
+
+	// filter possible duplicated capabilities from user's input
+	sCapabilitiesList = uniqueCaps(sCapabilitiesList)
+
+	// filter conflicting caps between ADD and DROP
+	if builder.definition.SecurityContext != nil {
+		if builder.definition.SecurityContext.Capabilities != nil {
+			if builder.definition.SecurityContext.Capabilities.Add != nil {
```

Comment:
Can be simplified to one if statement:
```suggestion
        if builder.definition.SecurityContext != nil &&
		builder.definition.SecurityContext.Capabilities != nil &&
		builder.definition.SecurityContext.Capabilities.Add != nil {
```

Reply:
that makes sense, thanks